website/content/docs/install/cloud-auto-join.mdx

```diff
@@ -1,9 +1,8 @@
 ---
 layout: docs
-page_title: Cloud Auto-join
+page_title: Auto-join a Cloud Provider
 description: >-
-  Consul supports automatically joining a Consul datacenter using cloud metadata
-  on various providers.
+Configure servers to auto-join clusters on startup. Use the CLI or an agent configuration file to specify either an IP address or a cloud provider for the server to join.
```

Comment:
```suggestion
Configure agents to automatically join agents running in the cloud by specifying their address using compute node metadata for the cloud provider instead of an IP address. Use the CLI or an agent configuration file to configure cloud auto-join.
```

Reply:
```suggestion
Auto-join enables agents to automatically join other agents running in the cloud. Configure auto-join by specifying agent addresses using compute node metadata for the cloud provider instead of an IP address. Use the CLI or an agent configuration file to configure cloud auto-join.
```
For me, the first sentence in Jared's suggestion is kind of a doozy. Is this any better?